mirage/utils/siaf_interface.py

```diff
@@ -0,0 +1,126 @@
+"""SIAF interface module to support accessing SIAF information.
+
+This module provides ``mirage`` with functions to interface SIAF content via the pysiaf module.
+
+Authors
+-------
+
+    - Johannes Sahlmann
+
+Use
+---
+
+    This module can be imported and used with
+
+    ::
+
+        from mirage.utils import siaf_interface
+
+"""
+
+import numpy as np
+
+import pysiaf
+from pysiaf import iando
+from ..utils import rotations
+from ..utils import set_telescope_pointing_separated as set_telescope_pointing
+
+
+def get_siaf_information(instrument, aperture, ra, dec, telescope_roll):
+    """Use pysiaf to get aperture information
+
+    Parameters:
+    -----------
+    instrument : str
+        Instrument name.
+
+    aperture : str
+        Aperture name (e.g. "NRCA1_FULL")
+
+    Returns:
```

Comment:
You don't need a "returns" section if it does not return anything. FYI.

Reply:
Removed...